packages/manager/apps/telecom/src/app/telecom/pack/migration/building-details/migration-building-details.controller.js

```diff
@@ -99,6 +101,13 @@ export default class TelecomPackMigrationBuildingDetailsCtrl {
     }
   }
 
+  /* Check OTP reference value is valid */
+  isOtpRegexValid() {
+    this.otpReferenceValid = this.model.ptoReference.match(
```

Comment:
```suggestion
    this.otpReferenceValid = this.model.ptoReference?.match(
```
Normally if you are this.model.ptoReference === null or '' you will have false so it's same result for this if statement.